test/performance/specs/site-editor.spec.js

```diff
@@ -138,7 +138,9 @@ test.describe( 'Site Editor Performance', () => {
 			// canvas is ready, and we don't want it to affect the typing
 			// timings.
 			await page
-				.locator( '.edit-site-canvas-loader' )
+				.locator(
+					'.edit-site-canvas-loader, .edit-site-canvas-spinner' // Spinner is used in the legacy editor.
```

Comment:
I think we can make the intention here a bit clearer for our future selves. 

```suggestion
					// Spinner was used instead of the progress bar in an earlier version of the site editor.
					'.edit-site-canvas-loader, .edit-site-canvas-spinner'
```